cmd/feed-ingress/main.go

```diff
@@ -132,6 +156,9 @@ func init() {
 	flag.DurationVar(&nginxConfig.UpdatePeriod, "nginx-update-period", defaultNginxUpdatePeriod,
 		"How often nginx reloads can occur. Too frequent will result in many nginx worker processes alive at the same time.")
 	flag.StringVar(&nginxConfig.AccessLogDir, "access-log-dir", defaultAccessLogDir, "Access logs direcoty.")
+	flag.StringVar(&gorbEndpoint, "gorb-endpoint", defaultGorbEndpoint, "Define the endpoint to talk to gorb for registration.")
+	flag.StringVar(&ingressInstanceIP, "ingress-instance-ip", defaultIngressInstanceIP,
+		"Define the ingress instance ip, the ip of the node where feed-ingress is running.")
```

Comment:
Isn't this specific to gorb/ipvs? We should prefix it accordingly.

Reply:
it isn't specific to ipvs, but only used in the gorb package at the moment. We could extract the instance ip lookup to an adapter that would decide whether to use it if provided or use the AWS API to lookup the instance ip otherwise. Then refactor gorb, elb and alb packages to make use of it

Reply:
ok, so we made if specific for now for simplicity and especially given the parallel stream of work that is already touching alb and elb packages. We can review that later on